activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -18,6 +20,16 @@ module FinderMethods
     #   Person.find([1])        # returns an array for the object with ID = 1
     #   Person.where("administrator = 1").order("created_on DESC").find(1)
     #
+    # ==== Find a composite primary key model
+    #   TravelRoute.primary_key = [:origin, :destination]
+    #   TravelRoute.find("Ottawa", "London") # returns the object for the route from Ottawa to London
+    #   TravelRoute.find(["Paris", "Montreal"]) # returns an array the object for the route from Paris to Montreal
+    #
+    #   # returns an array of the objects for the routes from New York to Las Vegas and from New York to Portland
```

Comment:
What do you think about, instead of describing what this returns, showing what the object would look like? e.g.
```
 #   TravelRoute.find(["New York", "Las Vegas"], ["New York", "Portland"])
 #   => [#<TravelRoute id: 2, origin: "New York", destination: "Las Vegas">, #<TravelRoute id: 3, origin: "New York", destination: "Portland">]
```

Reply:
I agree. I initially followed the example for regular `find` usages from above but personally I find object examples much more friendlier. Changed!